### PR TITLE
feat: add initial content of the github template

### LINKED ISSUE #166: [Feature] - simple miranum github template
- [x] Description created
- [x] Reviewed by peer: @Hafflgav 
- [ ] Prioritised

# Description

The objective of this issue is to create a simplified GitHub template for users looking to start a new project using Miranum. This template should be structured to facilitate an easy-to-follow, beginner-friendly initiation to the Miranum ecosystem, focusing on the key features of Miranum Connect - creating a Worker, correlating a message, and starting a process.

## Sub-Tasks
 
```[tasklist]
### Tasks
- [x] **Structure the template**
- [x] **Creating a worker**
- [x] **Correlate a message**
- [x] **Start a process**
- [x] **Documentation**: Write a README (don't forget to mention in miranum-docs)
```

### PR DESCRIPTION
**Pull request by rami-nk**
**Description**
This PR sets up the initial content of the github template using miranum-connect's core features.

**Checklist**

* Code is easy to understand
* No obvious errors or bugs
* Tests were implemented
* Documentation was created

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,66 @@
...
+To use this template, follow these steps:
+
+1. Clone or download this repository to your local machine.
+2. Update the necessary resources to match the name and description of your project:
+    - [README](./README.md) (title, description)
+    - [POM](./pom.xml) (artifact name, id, description)
+    - [Worker](src/main/java/io/miragon/example/MyWorker.java) (rename, implement)
+    - [Message Correlation](src/main/java/io/miragon/example/MyMessageCorrelation.java) (rename, implement)
+    - [Process Start](src/main/java/io/miragon/example/MyProcessStart.java) (rename, implement)
+3. Ensure that you have a running instance of Camunda Platform 7.
```
Comment (Hafflgav) [inline README.md:30]: ⛏️ I would also mention, that the C8 dependencies are included in the POM file as well and how a user can switch to C8.

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Camunda 7
```
Comment (Hafflgav) [inline src/main/resources/application.yaml:1]: ⛏️ Maybe add c8 properties to the application.yaml as comments as well. 
💭 I would briefly mention their existence in the README.

Comment (Hafflgav) [reply src/main/resources/application.yaml:1]: 💭 Maybe also add the local and SaaS config for C8: 

Local: 

> zeebe.client:
  broker:
    gateway-address: localhost:26500
  security:
    plaintext: true
    
Saas: 
> zeebe.client:
  cloud:
    region: bru-2
    clusterId: 
    clientId: 
    clientSecret:

**Review by Hafflgav — CHANGES_REQUESTED** [review]
Hey @rami-nk, great work with the template 🚀 
I left two comments. What do you think about them?

File: README.md (hunk 1 above)
Comment (rami-nk) [reply README.md:30]: Oh yess, forgot about that. But you are right!

**Review by Hafflgav — APPROVED** [review]
Yay! Great work! 🚀

**Pull request merged**